Description: Allow ISDN devices
 Users of ISDN dialup lines (such as me) cannot use wmppp as shipped,
 because it rejects to use any device except those beginning with `ppp'.
 .
 The patch attached solves this.
 .
 Originally appeared in version 1.3.0-5.  Converted to quilt for version
 1.3.0+20120605-1.
Author: Juan Cespedes <cespedes@debian.org>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=97509
Last-Update: 2014-09-11

--- a/wmppp/wmppp.c
+++ b/wmppp/wmppp.c
@@ -310,10 +310,15 @@
 					usage();
 					exit(1);
 				}
+				/* following removed to allow experiments with
+				 * new devices, i.e. ippp
+				 */
+#if 0
 				if (strncmp(argv[i+1], "ppp", 3)) {
 					usage();
 					exit(1);
 				}
+#endif
 				active_interface = argv[i+1];
 				i++;
 				break;
